docs/sphinx/using/backends/hardware.rst

```diff
@@ -112,6 +112,112 @@ the ``shots_count`` is set to 1000.
 
 To see a complete example for using IonQ's backends, take a look at our :doc:`Python examples <../examples/examples>`.
 
+Anyon Technologies/Anyon Computing
+==================================
+
+.. _anyon-backend:
+
+Setting Credentials
+```````````````````
+
+Programmers of CUDA-Q may access the Anyon API from either
+C++ or Python. Anyon requires a credential configuration file with username and password. 
+The configuration file can be generated as follows, replacing
+the ``<username>`` and ``<password>`` in the first line with your Anyon Technologies
+account details. The credential in the file will be used by the Cuda-Q to login Anyon quantum services 
+and will be updatd by Cuda-Q with obtained api_token and refresh_token. 
+Note, the credential line will be deleted in the updated configuration file. Please note that 
```

Comment:
Does this mean that the cleartext username/password are replaced with the "id_token" and "refresh_token" when someone runs any CUDA-Q program, or only if they manually run the `curl` command below?

Also, is the "Please note that" comment at the end of the line supposed to have something after it? I think it is an incomplete sentence right now.

Reply:
good catch. The "please note" should be deleted. Let me commit that change. 
The line is supposed to let the users know that their cleartext username/password will be deleted from the `.anyon_config` file and replaced with lines for `id_token` (in the file the keyword is `key`), `refresh_key` after the first run of CudaQ on the user side. Basically, every time CudaQ runs, it will check if it has the correct credentials (tokens or username/password). If it doesn't see tokens, it will POST the username/password to get tokens from backend ("/login"). After successful authentication of the username/password, tokens returned from the post request will be used to update the .anyon_config with tokens and CudaQ will interact with remote backend using these tokens. It minimizes the frequency of communications between frontend and backend using cleartext username/password as it will only be used once for login. `refresh_key` is the key used for requesting new `id_token` (`key`) if needed in the future. `id_token` (`key`) is used for CudaQ to submit and retrieve jobs with backend.